content/cli/hub.md

```diff
@@ -73,7 +73,7 @@ The CLI will need to be restarted, or refreshed, when changes are made on the Hu
 
 ## Theme Styles
 
-Theme styles correspond to the ability to change the terminal "skin" (i.e. coloring of the `menu`, `commands`, `data source`, `parameters`, `information` and `help`), as well as the chart and table styles.
```

Comment:
@Meg1211 don't just remove it without making sure sentence still makes sense - in this case "CLI skin"